src/main/java/com/github/distributedtracing/ContinuationLocalExecutorService.java

```diff
@@ -0,0 +1,115 @@
+package com.github.distributedtracing;
```

Comment:
It's the restore(state) method that's used to propagate state, it can accept any state really, as long as it has a reference to it.